app/jobs/scheduled/ignored_users_summary.rb

```diff
@@ -0,0 +1,42 @@
+module Jobs
+  class IgnoredUsersSummary < Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      return unless SiteSetting.ignore_user_enabled
+
+      params = {
+        threshold: SiteSetting.ignored_users_count_message_threshold,
+        gap_days: SiteSetting.ignored_users_message_gap_days,
+        coalesced_gap_days: SiteSetting.ignored_users_message_gap_days + 1,
+      }
+      user_ids = DB.query_single(<<~SQL, params)
+        SELECT ignored_user_id
+        FROM ignored_users
+        WHERE COALESCE(summarized_at, CURRENT_TIMESTAMP + ':coalesced_gap_days DAYS'::INTERVAL) - ':gap_days DAYS'::INTERVAL > CURRENT_TIMESTAMP
+        GROUP BY ignored_user_id
+        HAVING COUNT(ignored_user_id) >= :threshold
+      SQL
+
+      User.where(id: user_ids).find_each { |user| notify_user(user) }
+    end
+
+    private
+
+    def notify_user(user)
+      params = SystemMessage.new(user).defaults.merge({ ignores_threshold: SiteSetting.ignored_users_count_message_threshold })
+      title = I18n.t("system_messages.ignored_users_summary.subject_template")
+      raw = I18n.t("system_messages.ignored_users_summary.text_body_template", params)
+
+      PostCreator.create(
```

Comment:
ℹ️  I was tempted to wrap these two statements into a transaction, but then I thought if the Post creation, then it will all fail together.